helm/cert-manager-app/values.yaml

```diff
@@ -1,5 +1,4 @@
 name: cert-manager
-namespace: kube-system
```

Comment:
There's one in subchart too https://github.com/giantswarm/cert-manager-app/blob/master/helm/cert-manager-app/values.yaml#L2